src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -137,6 +139,8 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
             $this->outOfSyncFundFound = true;
         }
 
+        $this->entityManager->flush();
```

Comment:
is there a reason for adding this flush? Doesn't look like it will do any harm to performance or anything else but would be good to know. Afaik we're not removing a flush anywhere in this PR that would mean we need to add this. Think it's OK to add if it's just on a belt-and-braces principle.

Reply:
Having added the comment that's now on `Adapter::addAmount()`, I tried to trace back its direct callers and their callers to check that everything did in fact flush at some point. This was one place I couldn't find it doing so. And it's been so long since we ran an out of sync funds _correction_ that it seems quite possible we could have broken it at some point – so it seemed safer to add a flush to be sure persistence happens where we intended.